src/RefreshManager.m

```diff
@@ -486,6 +486,7 @@ -(void)refreshFeed:(Folder *)folder fromURL:(NSURL *)url withLog:(ActivityItem *
         if (![theLastUpdateString isEqualToString:@""])
         {
             [myRequest addRequestHeader:@"If-Modified-Since" value:theLastUpdateString];
+            [myRequest addRequestHeader:@"A-IM" value:@"feed"];
```

Comment:
Shouldn't it be `@"feed, gzip"` ? Our network stack is able to handle compressed response data.